R/get-osmdata.R

```diff
@@ -305,7 +305,7 @@ get_metadata <- function (obj, doc) {
 
 is_datetime <- function (x) {
 
-    ptn <- "[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}[A-Z]"
+    ptn <- "^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}[A-Z]$"
```

Comment:
Technically that's okay, but leaving in former state will allow the function to be more generally adapted to other cases. All timestamps returned as part of the OSM data set indeed have the form of the latter suggestion, but timestamps are, for example, also returned by the status queries - have a look at `overpass_status()`. The former version is highly specific anyway, and will allow timestamps to be extracted from general strings via regex matches if we need to. So i'd suggest leaving as was, unless you've encountered issues with that form?

Reply:
This new function just mimicry the behaviour of check_datetime https://github.com/ropensci/osmdata/blob/5c44982863b2e70fa4b99c219125aae457e66524/R/opq.R#L127-L152 (which can be simplified using regexp for a more ecological implementation) and is only used in get-osmdata.R:get_metadata(). If it has to be used in other places to match datetimes in a string, I would rename the function to contain_datetime. As you wish.

Reply:
Yeah, sound good, so we'll leave as is. Thanks!